### PR TITLE
Chore: Remove logs and warnings

### PR DESCRIPTION
**Pull request by lmuntanerDfinity**
# Motivation

Clean test logs and fix lint warnings.

# Changes

* Mock some dependencies causing logs in test.
* Fix some type and lint warnings.

# Tests

No new tests.

### CODE REVIEW

**Comment by lmuntanerDfinity** [comment]
@peterpeterparker @mstrasinskis please review

File: frontend/svelte/src/lib/components/ui/Tooltip.svelte [hunk 1]
```diff
@@ -2,7 +2,7 @@
   /** Used in aria-describedby */
   export let id: string;
   export let text = "";
-  export let noWrap: boolean = true;
+  export let noWrap: boolean = false;
```
Comment (peterpeterparker) [inline frontend/svelte/src/lib/components/ui/Tooltip.svelte:5]: Just to be sure, changing this default behavior still works for the UI right?

**Review by peterpeterparker — APPROVED** [review]
LGTM just one small question, thanks a lot

File: frontend/svelte/src/lib/components/ui/Tooltip.svelte (hunk 1 above)
Comment (lmuntanerDfinity) [reply frontend/svelte/src/lib/components/ui/Tooltip.svelte:5]: Yes, I checked. I was the one that needed this and messed it up with the styles when added this prop.

Comment (peterpeterparker) [reply frontend/svelte/src/lib/components/ui/Tooltip.svelte:5]: Super 👍

**Pull request merged**